image/image.go

```diff
@@ -11,6 +11,18 @@ import (
 
 var validHex = regexp.MustCompile(`^([a-f0-9]{64})$`)
 
+// Image stores the image configuration.
+// ID a unique 64 character identifier of the image
```

Comment:
I moved the comments into the structure. The doc will be slightly different, comments will be printed with the struct and then the comment about struct.